fixtures/events/performance_problems/n-plus-one-in-django-index-view.json

```diff
@@ -168,7 +168,7 @@
       "same_process_as_parent": true
     },
     {
-      "timestamp": 1661957871.418989,
+      "timestamp": 1661959871.418989,
```

Comment:
Why the timestamp change here?

Reply:
That span wasn't slow enough to be detected by the slow span detector